tasks/docs.rb

```diff
@@ -168,11 +168,11 @@ def properties_list(properties)
       end
     end
 
-    def special_properties(name, data)
+    def special_properties(name)
       properties = {}
 
       # these package properties support passing arrays for the package name
-      properties["common_resource_functionality_multiple_packages"] = true if %w{yum_package apt_package zypper_package homebrew_package dnf_package pacman_package homebrew_package}.include?(name)
+      properties["common_resource_functionality_multiple_packages"] = true if %w{snap_package dpkg_package yum_package apt_package zypper_package homebrew_package dnf_package pacman_package homebrew_package}.include?(name)
```

Comment:
I'm curious the difference between this and `"properties_multiple_packages"` 🤔

Reply:
It looks like we have some duplicate yaml items there. We should be setting both the same and I updated that. Longterm we should nuke one of those.